### PR TITLE
Continued with magic string issue

### PR DESCRIPTION
**Pull request by RobinLaidig**
Should be finished now, I went through the whole clams code

### CODE REVIEW

**Review by matejetz — CHANGES_REQUESTED** [review]

**Comment by RobinLaidig** [comment]
Fixed it now, thanks for the clarification

File: src/app/server-dashboard/Servers.ts [hunk 1]
```diff
@@ -3,9 +3,11 @@ import { Server } from './IServer';
 export class Servers {
 
   private readonly defaultServer: string = 'http:\\localhost:80';
```
Comment (matejetz) [inline src/app/server-dashboard/Servers.ts:5]: i would suggest we follow the [boy scout rule](https://deviq.com/boy-scout-rule/) (here meaning to also rename previously misnamed vars)

File: src/app/app.service.ts [hunk 2]
```diff
@@ -50,6 +50,8 @@ export class AppService {
    */
   readonly KEY_LAST_SERVER_ID = 'lastServerID';
 
+  private readonly KEY_PROJECT_DIRECTORY = 'ClamsProjectsDirectory';
```
Comment (matejetz) [inline src/app/app.service.ts:53]: this is still missing a type annotation

**Review by matejetz — CHANGES_REQUESTED** [review]
minor things, free to be merged after changes are applied

**Comment by matejetz** [comment]
nice!

**Pull request merged**